tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -740,4 +740,45 @@ virtual void TearDown() {
                    });
     CGPathApply(pathWithFontSizeAndTransforms, &comparePathContext, comparePathToExpected);
     ASSERT_EQ(expectedElements.size(), comparePathContext.count);
-}
\ No newline at end of file
+}
+
+const NSString* c_localeName = @"en-us";
+TEST(CTFont, CopyLocalizedName) {
+    auto fontName = woc::MakeAutoCF<CFStringRef>(CFSTR("Metadata Test"));
+    StrongId<NSURL> testFileURL = __GetURLFromPathRelativeToModuleDirectory(@"/data/MetadataTest-Regular.ttf");
+    CFErrorRef error = nullptr;
+    EXPECT_TRUE(CTFontManagerRegisterFontsForURL((__bridge CFURLRef)testFileURL.get(), kCTFontManagerScopeSession, &error));
+    EXPECT_EQ(nullptr, error);
+
+    auto font = woc::MakeAutoCF<CTFontRef>(CTFontCreateWithName(fontName, 20, nullptr));
+    ASSERT_NE(nullptr, font);
+
+    CFStringRef actualLanguage = nullptr;
+    auto copyrightName = woc::MakeAutoCF<CFStringRef>(CTFontCopyLocalizedName(font, kCTFontCopyrightNameKey, &actualLanguage));
+    EXPECT_OBJCEQ(c_copyrightName, (__bridge NSString*)copyrightName.get());
+    ASSERT_NE(nil, actualLanguage);
```

Comment:
>actualLanguage [](start = 19, length = 14)

nit: if actualLanguage was nil, this would never run.
Given they are not dependent, They should be able to be evaluated regardless.